src/modules/visits/controllers/visits.controller.ts

```diff
@@ -94,9 +94,26 @@ export class VisitsController {
 		const visits = await this.visitsService.findAll(queryDto, {
 			userProfileId: user.getId(),
 		});
+
 		return visits;
 	}
 
+	@Get('space/:id/access-status')
```

Comment:
QUESTION: should this be the id or the slug of the space? What does the client have available?